src/Microsoft.DotNet.Interactive.Formatting/JsonElementExtensions.cs

```diff
@@ -1,9 +1,9 @@
-﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Text.Json;
 
-namespace Microsoft.DotNet.Interactive.Extensions
+namespace Microsoft.DotNet.Interactive.Formatting
```

Comment:
Does this actually need to be public? It seems unrelated to this package's purpose.

Reply:
I moved that, that api is used in 2 places but no need to be on formatting